src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -1135,26 +1135,40 @@ private void GenerateExposedToBackingColumnMapsForEntities()
         {
             foreach ((string entityName, Entity _) in _entities)
             {
-                try
+                GenerateExposedToBackingColumnMapUtil(entityName);
+            }
+
+            foreach ((string entityName, Entity _) in _linkingEntities)
```

Comment:
Is this a change from this PR?

Reply:
Yes. 

As part of building the query, when generating the output columns, `EntityBackingColumnsToExposedNames` is looked at. But, when linking entity details are not populated, the flow fails when generating the output columns.
Hence, populating linking entity details in this data structure.